provider/pkg/schema/gen.go

```diff
@@ -936,6 +942,22 @@ func (ctx *context) propertySpec(propName, resourceTypeName string, spec *jssche
 		}
 	}
 
+	if propName == "Tags" {
+		switch ctx.GetTagsStyle(typeSpec, spec) {
+		case TagsStyleUntyped:
+			if propertySpec.Description != "" {
+				propertySpec.Description += "\n\n"
+			}
+			propertySpec.Description += "No additional type information is available in the schema for this property, please refer to the Cloud Control documentation for the expected schema."
```

Comment:
do we really need to add this? And if we think we do, why not add it to all properties of type "Any"?

Reply:
Hmm, maybe not. The first version of the message was a little more specific - indicating it was likely either a string map or a list of key-value objects, but I wasn't happy that the wording was clear enough so simplified it back to this, but perhaps we take it out.

Reply:
Ok, it now adds something like this for all properties:

```
Search the [CloudFormation User Guide](https://docs.aws.amazon.com/cloudformation/) for `AWS::Oam::Link` for more information about the expected schema for this property.
```